inspirehep/base/templates/format/record/templates_from_macros.tpl

```diff
@@ -0,0 +1,48 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
```

Comment:
The name of this file would be better something like `conference_info_macros.tpl`